src/assets/mocks/project.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
Zastanawiam się, czy przypadkiem nie jest to za duży narzut czasowy z przygotowywania oddzielnych serwisów, oraz mocków. Zastanawiam się, czy np. wraz z wzrostem aplikacji, kiedy mamy np. 20, 30, czy więcej możliwych requestów lepszym rozwiązaniem, nie jest serwis apiary, czy jakikolwiek inny, pełniący funkcję dokumentacji, jak i serwera zwracającego ustalone jsony.